autoscoper/src/ui/VolumeListWidgetItem.cpp

```diff
@@ -0,0 +1,52 @@
+#include "ui/VolumeListWidgetItem.h"
+#include "ui/AutoscoperMainWindow.h"
+
+#include <QCheckBox>
+#include <QLabel>
+#include <QDockWidget>
+#include <QGridLayout>
+#include <QSpacerItem>
+
+#if defined(Autoscoper_RENDERING_USE_CUDA_BACKEND)
+#include <gpu/cuda/RayCaster.hpp>
+#elif defined(Autoscoper_RENDERING_USE_OpenCL_BACKEND)
+#include <gpu/opencl/RayCaster.hpp>
+#endif
+
+VolumeListWidgetItem::VolumeListWidgetItem(QListWidget* listWidget,QString name, AutoscoperMainWindow* main_window, std::vector< xromm::gpu::RayCaster*>* renderers) : QListWidgetItem(listWidget) {
+  this->name_ = name;
+  visible_ = true;
```

Comment:
```suggestion
```

This is redundant with the state already associated with the checkbox and RayCaster